packages/terra-clinical-header/src/Header.scss

```diff
@@ -1,7 +1,7 @@
 :local {
   .flex-header {
-    background-color: #f4f4f4;
-    border-bottom-color: #dedfe0;
+    background-color: var(--terra-header-background-color, #f4f4f4);
```

Comment:
Can we updates these to be name-spaced to terra-clinical?

Reply:
Based on other scss files in this repo, I omitted clinical from the name. I have no problem adding the name spacing here, just wanted to make sure we are aware we are breaking that pattern.

Reply:
oh no... IMO  that pattern that should have been followed for the clinical components.
@ryanthemanuel @mjhenkes thoughts?

Reply:
Yes, let's go ahead make this one terra-clinical.  The other ones probably should be as well, but that would be non-passive at this point.